common.blocks/progressbar/progressbar.bemhtml

```diff
@@ -4,11 +4,22 @@ block('progressbar')(
         return { val : this.ctx.val };
     }),
 
+    attrs()(function() {
+        return {
+            role : 'progressbar',
+            'aria-label' : this.ctx.ariaLabel,
+            'aria-labelledby' : this.ctx.ariaLabelledBy,
+            'aria-valuemin' : 0,
+            'aria-valuemax' : 100,
+            'aria-valuenow' : this.ctx.val || 0
```

Comment:
According to our docs `this.ctx.val` should always [be `Number`]. In other cases you should fix js-mode above or block's JS will fail as @sipayRT have mentioned.

[1]: https://en.bem.info/libs/bem-components/v2.0.0/desktop/progressbar/docs/#val